schedule/autoyast_mini_staging.yaml

```diff
@@ -0,0 +1,20 @@
+name:           autoyast_mini_staging@64bit-staging
```

Comment:
Does @64bit-staging dorce it to run only against x86-64?

Reply:
It does not force, it's just a side note, that the test suite should run only in staging intel arch.